File: LeetCodePracticeQuestions/In_Java/Easy/FinalValueofVariableAfterPerformingOperations.java
class Solution {
    public int finalValueAfterOperations(String[] operations) 
    {
      int X=0;  

      int len=operations.length;

      for(int i=0;i<len;i++)
      {
      if(operations[i].equals("X++") || operations[i].equals("++X"))
        X++;  

      if(operations[i].equals("X--") || operations[i].equals("--X"))
        X--;    
    }
   
    return X;
}
}

File: LeetCodePracticeQuestions/In_C/Easy/FinalValueofVariableAfterPerformingOperations.c
int finalValueAfterOperations(char** operations, int operationsSize) 
{
      int X=0;  

      int len=operationsSize;

      for(int i=0;i<len;i++)
      {
      if(strcmp(operations[i],"X++")==0 ||strcmp(operations[i],"++X")==0)
        X++;  

      if(strcmp(operations[i],"X--")==0 || strcmp(operations[i],"--X")==0)
        X--;    
    }
   
    return X; 
}
